openedx/core/djangoapps/safe_endpoints/middleware.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from rest_framework_jwt.authentication import BaseJSONWebTokenAuthentication
+
+
+log = logging.getLogger(__name__)
+
+
+class EnsureJWTAuthSettingsMiddleware(object):
+    """
+    Django middleware object that ensures the proper Permission classes
+    are set on all endpoints that use JWTAuthentication.
+    """
+    required_permission_classes = ()
```

Comment:
What are the required permission classes?  Seems like a lot of trouble to go through if we don't have any. ;)

Reply:
Once @douglashall merges his PR, I'll update this PR to include his new `JwtHasScope` permission class.